.dockerignore

```diff
@@ -0,0 +1 @@
+.gitignore
```

Comment:
Is this file still needed if we're not including the code repo in the built image?